### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by jecihjoy**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

**Comment by jecihjoy** [comment]
[Uploading Screencast from 06-04-2024 05:52:54 ALASIRI.webm…]()

**Review by njorocs — APPROVED** [review]
Good for merge

**Comment by ojwanganto** [comment]
Thanks @jecihjoy. Please re-attach the video for our eyes oh :)

File: src/types/index.ts [hunk 1]
```diff
@@ -426,15 +426,14 @@ export interface ProcedureOrderBasketItem extends OrderBasketItem {
   instructions?: string;
   previousOrder?: string;
   orderReason?: string;
-  scheduleDate?: string | Date;
-  // scheduleDate?: Date;
+  scheduledDate?: string | Date;
   commentsToFulfiller?: string;
   laterality?: string;
-  numberOfRepeats?: number;
+  numberOfRepeats?: string;
```
Comment (ojwanganto) [inline src/types/index.ts:432]: should this be a string?

**Pull request merged**